flake8_plugin_utils/plugin.py

```diff
@@ -142,14 +124,3 @@ def test_config(cls, config: TConfig) -> Iterator[None]:
             yield
         finally:
             del cls.config
-
-
-def check_noqa(line: str, code: str) -> bool:
```

Comment:
I'm not sure how to proceed about deleting `check_noqa`, `NOQA_REGEXP` and `NOQA_ERROR_CODE_REGEXP`. Technically they are part of the public API (though undocumented), but I doubt anyone uses them directly.
Bumping the major version would also be a problem because most dependent plugins are using "^1.x" range

Reply:
I will increase the minor version, nothing terrible should happen.